scalableminds/webknossos
# Improve visual separation of layer-specific sections in settings

This PR improves the visual separation of the layer-specific sections in the settings by adding light grey lines between them.

### URL of deployed dev instance (used for testing):
- https://layersectionseparation.webknossos.xyz

### Steps to test:
- open a tracing for a dataset with several layers
- open the settings
- the layer sections should be divided by light grey lines

### Issues:
- solves #4687

------
